posthog/models/performance/sql.py

```diff
@@ -114,7 +114,7 @@
 PERFORMANCE_EVENTS_TABLE_SQL = lambda: (
     PERFORMANCE_EVENTS_TABLE_BASE_SQL()
     + """PARTITION BY toYYYYMM(timestamp)
-ORDER BY (team_id, toDate(timestamp), session_id, pageview_id, timestamp)
+ORDER BY (team_id, toDate(timestamp), session_id, timestamp)
```

Comment:
this should definitely have a hash of the current url in we'll frequently query by url

maybe we could add resource type too since for analytics we'll probably offer "fetch" and "navigate" as the query types since folk will most probably want page load and API speeds for analytics